### ISSUE TITLE
Cannot start the game locally (ERR_INVALID_URL)

### ISSUE THREAD

**Issue by krystofNovotny** [pongstylin/tactics#78]
I am having trouble running the game locally. 

I run `npm install`, `npm start` and the game doesn't start.
I get an error message I'm not sure how to fix (see below).

I have updated node and npm to the latest version and tried to google a bit.
No luck so far, any help would be greatly appreciated.

version of node: v12.16.2
version of npm: 6.14.5
os: Ubuntu 18.04.4 LTS

Output of `npm start`:
```
master!tactics *> npm start

> tactics@0.10.7 start /home/krystof/fun/tactics
> NODE_ENV=development webpack --watch --hide-modules --config webpack.config.cjs & node --es-module-specifier-resolu
tion=node --experimental-modules --experimental-loader ./resolver.mjs --require dotenv/config src/server.js || true

(node:17688) ExperimentalWarning: The ESM module loader is experimental.
(node:17688) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any 
time
internal/modules/run_main.js:57
    internalBinding('errors').triggerUncaughtException(
                              ^

TypeError [ERR_INVALID_URL]: Invalid URL: http
    at onParseError (internal/url.js:257:9)
    at new URL (internal/url.js:333:5)
    at Loader.defaultGetFormat [as _getFormat] (internal/modules/esm/get_format.js:44:18)
    at Loader.getFormat (internal/modules/esm/loader.js:110:42)
    at Loader.getModuleJob (internal/modules/esm/loader.js:241:31)
    at async ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:44:17)
    at async Promise.all (index 1)
    at async link (internal/modules/esm/module_job.js:48:9) {
  input: 'http',
  code: 'ERR_INVALID_URL'
}
```

**Comment by pongstylin** [comment]
I just committed a fix to make the game compatible with Node v12.16.2.  Update and try again.

**Comment by krystofNovotny** [comment]
Awesome, I can confirm it's fixed now, thanks.
